src/core/js/models/adaptModel.js

```diff
@@ -236,9 +245,9 @@ define([
         },
 
         getAvailableChildren: function() {
-            return new Backbone.Collection(this.getChildren().where({
+            return this.getChildren().where({
```

Comment:
@moloko and I had a conversation about this. it might be worth just swallowing the repercussions and take advantage of the v3 release to just do this outright? that would be my preference even though it'll require everyone who uses these two functions to update their extensions.